src/csharp/examples/server/API/DoCoolStuff/CoolStuffModel.cs

```diff
@@ -0,0 +1,11 @@
+namespace server.Models
+{
+    public class CoolStuffModel
+    {
+        public string doubleCool { get; set; } = "default";
```

Comment:
the setter `= "default";` is useless when you have a constructor in the class which is setting it already

if this was intended then you would need an empty constructor so "default" wouldnt be wiped instantly
```csharp
public CoolStuffModel() {} // this wont set DoubleCool so it will keep whatever value it had set
```

or have no constructors in the whole class (C# defaults to empty constructor implicitly like the one mentioned above)

---
### Style:
C# Recommends using PascalCase for property names such as `DoubleCool`

Reply:
ah, misunderstood the error from the other models that don't have constructors. As far as the PascalCase, this needs to get serialized to the cli tester so it needs to eventually be camelCase. If this was a real life implementation I'd probably put in the work to Pascal here and camel there, but since it's just an example I'm inclined to keep it as is.